mindsdb/integrations/handlers/mindsdb_inference/README.md

```diff
@@ -0,0 +1,74 @@
+---
```

Comment:
As an aside for the future, I wonder if there is a way we can somehow avoid duplication between this readme and the one in docs.